lib/codelog/command/new.rb

```diff
@@ -30,6 +35,14 @@ def default_editor
         '${VISUAL:-${EDITOR:-nano}}'
       end
 
+      def change_file_timestamp
+        Time.now.strftime('%Y%m%d%H%M%S%L')
+      end
+
+      def change_file_name
+        @name.gsub(/(.)([A-Z])/, '\1_\2').downcase
```

Comment:
It would be nice to leave a comment explaining what this regex do.

Reply:
I will do it